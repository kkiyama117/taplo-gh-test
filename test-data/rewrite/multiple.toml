[remove_this]

[remove_this.subtable]
value = "should be removed"

[table]
rewrite_me = 2

[[table.arr]]
rewrite_my_value = 3

[[table.arr]]
rewrite_my_value = 3
