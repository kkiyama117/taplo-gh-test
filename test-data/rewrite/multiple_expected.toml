

[table]
rewritten = 2

[[table.arr]]
rewrite_my_value = 0

[[table.arr]]
rewrite_my_value = 1
